ros2topic/ros2topic/api/__init__.py

```diff
@@ -57,19 +57,20 @@ def __call__(self, prefix, parsed_args, **kwargs):
 def import_message_type(topic_name, message_type):
     # TODO(dirk-thomas) this logic should come from a rosidl related package
     try:
-        package_name, *message_name = message_type.split('/')
-        if not package_name or not message_name or not all(message_name):
+        package_name, middle_module, message_name = message_type.split('/')
+        if not middle_module:
+            middle_module = 'msg'
+        if not package_name  or not message_name:
             raise ValueError()
     except ValueError:
         raise RuntimeError('The passed message type is invalid')
 
     # TODO(sloretz) node API to get topic types should indicate if action or msg
-    middle_module = 'msg'
     if topic_name.endswith('/_action/feedback'):
         middle_module = 'action'
 
     module = importlib.import_module(package_name + '.' + middle_module)
```

Comment:
Perhaps we should just support any number of parts, e.g. `'.'.join([package_name] + message_name[:-1])`.

Reply:
Does it not generally make sense to unify this logic and have it in a central place? I know of at least one more other location where we duplicate the same logic:
https://github.com/ros2/ros2cli/blob/master/ros2service/ros2service/verb/call.py#L61-L72

Reply:
Yes. There's a TODO at the top of this function. I'm not sure what package it should belong in (or if a new package needs to be created.